templates/Dockerfile.template

```diff
@@ -0,0 +1,24 @@
+FROM {{ base_image }}
+
+ENV HOME=/home/ops \
+    CONDA_DIR=/opt/conda
+
+USER root
+RUN $CONDA_DIR/bin/conda install -y nb_conda_kernels
+
+USER ops
+RUN $HOME/verdi/bin/pip install 'ipykernel<5.0.0'
+
+RUN cd $HOME \
```

Comment:
Should comment out these since notebook_pge_wrapper should be installed already. Add comment that user can uncomment if they want the latest installed.